components/scream/tests/scream_p3/p3_init_vals.txt

```diff
@@ -0,0 +1,216 @@
+0 0 0.0 1079.543354958876 100.0 34903.91666666666 0.0 0.0 0.0 1.0 0.0 1000000.0 0.0 1000000.0 0.0 1000000.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 1388.888888888889 7.197210072907482 0.0 0.0 0.0 0.0 0.0 0.0 1.0 1.0 1.0 0.0 0.0 0.0 0.0 0.0 0.0 150.0

```

Comment:
Is it possible to include a 1-line header that describes the names of the entities? Maybe not for this PR, but to make folks' lives easier in the future.

Reply:
for sure, I made the change.